tests/hash/password_hash_001.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**Test password_hash blowfish algorithm */
+$password = "rasmuslerdorf";
+$salt = "Ajnbu298IRHUVa56XvDOzu";
+$cost = 15;
+$options =[
+    'cost' => 15,
+    'salt' => $salt,
+];
+
+$hashD = @password_hash($password,PASSWORD_DEFAULT,$options);
+$hashB = @password_hash($password,PASSWORD_BCRYPT,$options);
+
+echo "PASSWORD_DEFAULT : $hashD\n";
+echo "PASSWORD_BCRYPT : $hashB\n";
+?>
```

Comment:
We usually don't put the ```?>``` closing tags in tests, remove it from all the added ones, please. In ```.phpt``` files in the PHP source code they do it to separate the code execution from the following lines with the expected results, we don't need to do that.